src/cli-constant.js

```diff
@@ -167,6 +167,12 @@ const detailedOptions = normalizeDetailedOptions({
     default: ".prettierignore",
     description: "Path to a file with patterns describing files to ignore."
   },
+  "indent-styles": {
+    type: "boolean",
+    category: CATEGORY_FORMAT,
+    forwardToApi: true,
+    description: "Indent adjacent CSS Rules based on CSS selector hierarchy"
+  },
```

Comment:
Sorry for the nitpick, but the options here should be sorted alphabetically.

Reply:
No problem changing this of course but I don't quite understand which part you're referring to...I guess you're referring to the "type", "category", "forwardToApi" "description" but other options ("insert-pragma") don't follow this ordering, so could you confirm that I should change that? The string "indent-styles" is alphabetically between "ignore-path" and "insert-pragma", so I guess you don't mean that one?

Reply:
Sorry, I was very unclear. This is what I meant: https://github.com/prettier/prettier/blob/c0fa47d2d26eb211156a1628d9370efb36c6afc0/src/cli-constant.js#L77

Reply:
Hmm... sorry, I feel really dumb because I must be missing something. The string "**ind**ent-styles" is already alphabetically between "**ign**ore-path" and "**ins**ert-pragma" is it not?

```js
> ["indent-styles", "ignore-path", "insert-pragma"].sort()
["ignore-path", "indent-styles", "insert-pragma"]
```

Very sorry about this because I'm trying to adjust the code but I'm a bit confused by this.

Reply:
> sorry, I feel really dumb because I must be missing something.

No, it is I who is dumb – can't even do the alphabet anymore … sorry, you are correct. Nothing to do here :)